mm2src/mm2_main/src/lp_price.rs

```diff
@@ -224,8 +225,8 @@ async fn try_price_fetcher_endpoint(
     })
 }
 
-/// consume try_price_fetcher_endpoint result here using different endpoints.
-/// return price data on Success or None on failure.
+/// Consume `try_price_fetcher_endpoint` result here using different endpoints.
+/// Return price data on `success` or None on `failure`.
```

Comment:
`success` and `failure` are not required to be highlighted

Reply:
Done